site/content/docs/manifest/rd-web-service.ja.md

```diff
@@ -66,6 +67,9 @@ http:
     timeout: 10s
```

Comment:
Adding this comment for the line.51 (since we don't have any diffs in that line). We also need to update the "http セクションは Service と AWS App Runner のマネージドロードバランサの連携に関するパラメーターを含みます。" as the original English docs describes here as the following.

`The http section contains parameters related to the managed load balancer.`

Reply:
I'm sorry I overlooked...